Updates

JakeDaBoss opened the pull request:
- Changelog update.

JakeDaBoss committed:
Merge remote-tracking branch 'refs/remotes/origin/dev' into Oliver-Test-Branch

JakeDaBoss committed:
Changelog 1.

JakeDaBoss committed:
Changelog 2.

JakeDaBoss commented:
Someone review please. @BlueNexus @Lykayos @WrongEnd @Nightspark43

BlueNexus commented on html/changelogs/.all_changelog.yml:
```diff
@@ -3394,3 +3394,11 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - bugfix: Fixed Medical Examiner.
   - bugfix: Fixed upper Command power issues.
+2016-10-02:
+  N3X15:
```
I don't recall N3X15 working on this repo. Looks like _someone_ forgot to edit the name when they made their changelog. _looks at Oliver_

JakeDaBoss replied:
I had to make Lyka's, okay? :/

JakeDaBoss committed:
Fix for Changelog.

JakeDaBoss commented:
Done @BlueNexus

JakeDaBoss committed:
Fix to button not being linked in crewbriefing.

JakeDaBoss committed:
Fix to area mismatch.

JakeDaBoss committed:
Fixes to HoP's office.

JakeDaBoss committed:
Massive changes

- Thrown some verbs around, reorganising some of the mod's permissions.
- Added some new commands.
- Fixed an unwired APC in the morgue.

Compiles fine. Fixes #27

JakeDaBoss committed:
More fixes to HoP's office.

JakeDaBoss committed:
Added Admincryo.

JakeDaBoss committed:
Fixes Morgue stairs.

JakeDaBoss committed:
Changelog.

JakeDaBoss committed:
Fix to z-level defines.

JakeDaBoss committed:
Changelog update again.

BlueNexus commented:
I suggest asking #codershuttle about using qdel instead of del, before this is merged.

BlueNexus commented on code/modules/admin/verbs/admincryo.dm:
```diff
@@ -0,0 +1,161 @@
+					if(O.target && istype(O.target,/datum/mind))
+						if(O.target == M.mind)
+							if(O.owner && O.owner.current)
+								O.owner.current << "\red You get the feeling your target is no longer within your reach. Time for Plan [pick(list("A","B","C","D","X","Y","Z"))]..."
```
Depreciated. Changed \red to `<span class="warning">`

BlueNexus commented on code/modules/admin/verbs/admincryo.dm:
```diff
@@ -0,0 +1,161 @@
+				//Make an announcement and log the person entering storage.
+				//frozen_crew += "[M.real_name]"
+
+				message_admins("\blue [key_name_admin(usr)] has admin cryoed [key_name(M)]")
```
Ditto, but for notice, not warning.

JakeDaBoss commented:
I'll fix this this afternoon.

JakeDaBoss commented:
@BlueNexus Always use qdel.

BlueNexus commented:
Please correct all instances of del, then.

JakeDaBoss committed:
Fixes, additions, more of that jazz.

JakeDaBoss commented:
Fixes #27

JakeDaBoss committed:
More updates.

Fixes #32
Fixes #27

JakeDaBoss committed:
Sneak update to Captain's Office.

JakeDaBoss committed:
Fixes to map.

JakeDaBoss committed:
Fixes.

JakeDaBoss committed:
ID Fixes.

JakeDaBoss committed:
Added Med Laptop to Medbay lobby.

JakeDaBoss committed:
Changelog.

JakeDaBoss committed:
Added disposal loop.

JakeDaBoss committed:
Oops.

JakeDaBoss committed:
Oops #2
